less/navbar.less

```diff
@@ -11,19 +11,21 @@
   overflow: visible;
   margin-bottom: @baseLineHeight;
 
+  &.navbar-no-margin-bottom {
+    margin-bottom: initial;
```

Comment:
Don't know - I just copied this file back from the Ace repo so someone made a change to it there and this just brings the repo in line with Ace.